wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -507,43 +507,51 @@ <h2>Dropdown buttons</h2>
                     <div class="dropdown dropdown-button match-width">
                         <input type="button" value="drop down" class="button" />
                         <div class="dropdown-toggle icon icon-arrow-down"></div>
-                        <ul role="menu">
-                            <li><a href="#">items should not exceed button width</a></li>
-                            <li><a href="#">item 2</a></li>
-                        </ul>
+                        <nav aria-label="dropdown menu example with matched width">
```

Comment:
I would suggest not having a landmark for all of the ones here, since we don’t know in advance the context in which those examples would be used.